### PR TITLE
Profile pages

### PR DESCRIPTION
**Pull request by smathis122**
Fixed merge conflicts that occurred within main

set up the profile pages to show what items the user has hosted

### CODE REVIEW

**Review by casejade37 — APPROVED** [review]
Good work as always!

File: code/Frontend/react-frontend/src/pages/ViewItem.js [hunk 1]
```diff
@@ -9,14 +9,45 @@ import Button from "react-bootstrap/esm/Button";
...
+        <p>You are a host!</p>
+        {/* Display the user's profile information */}
+      </div>
+    );
+  } else {
+    return (
+      <div>
+        <h1>Profile Page</h1>
+        <p>Welcome, {username}!</p>
+        <p>I don't know what you are!</p>
```
[the remark this answers is not in the document]
Comment (smathis122) [reply code/Frontend/react-frontend/src/pages/ViewItem.js:36]: Haha I took this from Ahnaf's section but combined it with the stuff I had been working on

File: code/Frontend/react-frontend/src/pages/ViewItem.js [hunk 2]
```diff
@@ -9,14 +9,45 @@ import Button from "react-bootstrap/esm/Button";
...
+        {/* Display the user's profile information */}
+      </div>
+    );
+  }
+}
+
+function View() {
```
[the remark this answers is not in the document]
Comment (smathis122) [reply code/Frontend/react-frontend/src/pages/ViewItem.js:43]: Yeah I get what you mean by that. I can update it in the future

**Review by saahilv — APPROVED** [review]
Looks good, please merge

**Pull request merged**